app/app.js

```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
+  $newChicken.find('#4').text("Average Weight: " + this.weight);
   $newChicken.find('.publishedOn').text("Published on " + this.publishedOn +" (" + daysAgo +" days ago)");
   $newChicken.find('.description').html(this.description);
-  $newChicken.find('.headShot').html(this.image);
+  $newChicken.find('.head-shot img').attr('src',  this.image);
 
   return $newChicken;
 };
 
+//passes constructed objects to allChickens
 rawChickenData.forEach(function(ele){
   allChickens.push(new Chicken(ele));
 });
 
+function populateChickenFilter (chickenName){
+  var optionTag = '<option value="' + chickenName+ '">' + chickenName + '</option>';
+  if($('chicken-filter opt[value="' + chickenName +'"]').length === 0){
+    $('.chicken-filter').append(optionTag);
+  }
+}
+
+function populateTemperFilter (chickenTemper){
+  var optionTag = '<option value="' + chickenTemper+ '">' + chickenTemper + '</option>';
+  if($('.temper-filter option[value="' + chickenTemper + '"]').length === 0) {
+    $('.temper-filter').append(optionTag);
+  }
+}
+
 allChickens.forEach(function(a){
   $('#chickens').append(a.toHtml());
+  populateChickenFilter(a.title);
+  populateTemperFilter(a.temperament);
 });
 
-$('img').css("width", "180px");
+// filter by title
+function chicFilter(){
+  $('.chicken-filter').change(function(){
+    $('article').hide();
+    var selectedValue = $(this).val();
+    console.log(selectedValue);
+    if(selectedValue){
+      $('article[data-name="' + selectedValue + '"]').fadeIn(800);
+    }else{
+      $('article').not('.template').show();
+    }
+  });
+}
+chicFilter();
 
-$('.template').hide();
+// filter by temperament
+function tempFilter(){
+  $('.temper-filter').change(function(){
+    $('article').hide();
+    var selectedValue = $(this).val();
+    console.log(selectedValue);
+    if(selectedValue){
+      $('article[data-temper="' + selectedValue + '"]').fadeIn(800);
+    }else{
+      $('article').not('.template').show();
+    }
+  });
+}
+tempFilter();
 
-$('#hamburger').click(function(){
-   $('.main-nav').fadeToggle("slow");
-});
-$('nav').find('li:first').on('click', function(){
-  $('#chickens').hide();
-  $('#about').fadeIn().css({"height": "600px"})
-});
-$('nav').find('li:nth-child(2)').on('click', function(){
-  $('#about').hide();
-  $('#chickens').fadeIn()
+function teaser(){
+  console.log('in the fuction');
+  var hiddenContent = $('.description *:nth-of-type(n+2)')
+ hiddenContent.hide();
+ $('.read-on').click(function(e) {
+   e.preventDefault();
+   console.log('you clicked');
+   if($(this).text() === 'Read more') {
+    $('.description').find('p').fadeIn();
+     $(this).text('Show Less');
+   } else if ($(this).text() === 'Show Less'){
+     hiddenContent.hide();
+     $(this).text('Read more');
+   }
+ });
+}
+teaser();
+
+$(document).ready(function(){
+  //show only About section
+  $('nav').find('li:first').on('click', function(){
+    $('#chickens').hide();
+    $('#about').fadeIn().css({"height": "600px"});
+  });
```

Comment:
No prevent default in this click.
Also, can you see a way to make these two click handlers actually work for both?

I noticed that when I clicked on these, the filters for the articles were still showing. Do you know why?